MarketAnal.vcxproj

```diff
@@ -60,7 +60,7 @@
   </PropertyGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
     <ClCompile>
-      <AdditionalIncludeDirectories>%(AdditionalIncludeDirectories);$(Qt_INCLUDEPATH_);./InvestingCPR;</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(ProjectDir)src;$(ProjectDir)include;$(ProjectDir);%(AdditionalIncludeDirectories);$(Qt_INCLUDEPATH_);./InvestingCPR</AdditionalIncludeDirectories>
```

Comment:
It is changing structure. that is why it needs to be merged before add submodule.

Reply:
.